### PR TITLE
Emergency Reboot Boards notify the ghosts of dead borgs

### PR DESCRIPTION
**Pull request by Super3222**
Refer to issue https://github.com/yogstation13/yogstation/issues/2032.

#### Changelog

:cl:
rscadd: Borgs are notified when they're being revived.
/:cl:

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
doesn't it already pull the ghost into the body immediately

**Comment by Super3222** [comment]
they are. am i looking at the wrong one?

File: code/game/objects/items/robot/robot_upgrades.dm [hunk 1]
```diff
@@ -87,6 +87,9 @@
 		for(var/mob/dead/observer/ghost in player_list)
 			if(ghost.mind && ghost.mind.current == R)
 				R.key = ghost.key
+				ghost << "<span class='boldnotice'>An enemrgency reboot module has been installed \
```
Comment (Cruix) [inline code/game/objects/items/robot/robot_upgrades.dm]: Emergency

Comment (ShadowDeath6) [reply code/game/objects/items/robot/robot_upgrades.dm]: enemrgency

**Comment by Mrcreepssss** [comment]
i was once a ghosted borg and it didn't pull me in...

edit: also, do they get a bit of time to react before it activates too? like with defibs and cloners?

**Comment by Super3222** [comment]
Jumping back into your body should be enough. Unless the roboticist hasn't fixed enough of the damages you revive you.

**Comment by Mrcreepssss** [comment]
so with a reboot card you can jump back in any time and be alive?

**Pull request merged**